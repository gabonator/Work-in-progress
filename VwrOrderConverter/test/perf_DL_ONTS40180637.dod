X|47675934||ONTS40180637|20181211|63073242|1|45.00|55.00|.00|.00|45.00|54.50|
|2.00|.00|22.50|27.23|21.0|||||Kartac na lahve s PH drzakem (silon)pr.50|VIT224416428||VIT224416428|ks|274|