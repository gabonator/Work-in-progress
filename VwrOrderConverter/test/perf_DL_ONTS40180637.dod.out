BEGIN;ONTS40180637;Perfect;VWR;ORDERS;20181211;102511;HEAD;ONTS40180637;20181211;;CZK;SOLDTO;26002813;Perfect Distribution a.s. (Třinec)SHIPTO;26006315;Perfect Distribution a.s. (Třinec);;;;Konská 198;;;;;Třinec;739 61;CZ;libuse.styskalova@pfd.agel.cz;+420 606 784 687LINE;000001;;KAKO4416428;Kartac na lahve s PH drzakem (silon)pr.50;2.000;;20181211;22.50;45.00END;ONTS40180637